sdk/servicebus/microsoft-azure-servicebus/pom.xml

```diff
@@ -87,5 +87,11 @@
       <version>4.13-beta-3</version> <!-- {x-version-update;junit:junit;external_dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+        <groupId>org.apache.httpcomponents</groupId>
+        <artifactId>httpclient</artifactId>
+        <version>4.5.8</version>
```

Comment:
Hey, our versioning story is managed by a series of scripts to keep dependency versions aligned. Can you add this annotation and follow the instructions to update to the proper version?
https://github.com/Azure/azure-sdk-for-java/blob/master/CONTRIBUTING.md#versions-and-versioning

`<!-- {x-version-update;org.apache.httpcomponents:httpclient;external_dependency} -->`